requirements/connectors/vectara.in

```diff
@@ -1,3 +1,4 @@
 -c ../common/constraints.txt
 
 requests
+aiofiles
```

Comment:
Now that we're using httpx, that should be listed here. 
```suggestion
aiofiles
httpx
```

Reply:
Adding it.

Reply:
addd